src/xcode/ENA/ENA/Source/Scenes/TicketValidation/TicketValidationCoordinator.swift

```diff
@@ -101,7 +101,33 @@ final class TicketValidationCoordinator {
 	}
 
 	private func showResultScreen(for result: TicketValidationResult) {
+		let viewModel: TicketValidationResultViewModel
 
+		switch result.result {
+		case .OK:
```

Comment:
who defines these enum cases? I find their naming a little bit meaningless. Can we change that or comes this from the CertLogic or sth. similar?

Reply:
The case names come from the server response. I moved the cryptic names to the rawValue now and gave the cases clearer names.